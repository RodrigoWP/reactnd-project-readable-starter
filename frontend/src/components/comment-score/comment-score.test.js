'use strict'

import React from 'react'
import { shallow } from 'enzyme'
import { CommentScore } from '.'

it('Should render correctly without props', () => {
  const component = <CommentScore icon={''} />

  expect(shallow(component)).toMatchSnapshot()
})
