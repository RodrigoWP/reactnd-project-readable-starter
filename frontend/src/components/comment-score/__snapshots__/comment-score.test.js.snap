// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly without props 1`] = `
<ScoreIcon
  count={0}
  icon=""
/>
`;
